HealthcareSystem/Backend/Model/Manager/Equipment.cs

```diff
@@ -1,17 +1,35 @@
 /***********************************************************************
- * Module:  Equipment.cs
+ * Module: Equipment.cs
  * Author:  Jelena Budisa
- * Purpose: Definition of the Class Equipment
+ * Purpose: Definition of the Class ConsumableEquipment
  ***********************************************************************/
 
 using System;
-
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace Model.Manager
 {
     public class Equipment
```

Comment:
I like that you got rid of the hierarchy, it wasn't really adding any value to the model I think.